RESTfulAPI.md

```diff
@@ -282,3 +285,5 @@ GET /companypercentage/alexa
 ## API (that you may need)
 
 TLS/SSL global collection, can be fetched and interpolated:  https://scans.io/
+
+@vecna: Not sure I understand. Can you give an example?
```

Comment:
@dropmeaword The mention works only in the comment messages, not in the commit (you have proposed changed that contain the @vecna messages, I'm not gonna to accept it).